_layouts/post.html

```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+---
+
+{% include back-to-top.html %}
+
+<p><a href="{{site.baseurl}}/blog">Blog</a> <i class="fas fa-chevron-right"></i> {{ page.title }}</p>
+
+
+<div class="home post-page">
+  {{ page.date | date: '%B %d, %Y' }} 
+
+  <h1>{{ page.title }}</h1>
+  
+  <div class="byline-wrapper">
+    {% assign author = site.data.authors[page.author] %}
+    {% if author.emailhash %}
+    <img class="headshot" src="https://www.gravatar.com/avatar/{{ author.emailhash }}">
+    {% endif %}
+    <p class="byline">By {{ author.name }}</p>
```

Comment:
Do you think we should add author bio somewhere here too? Its declared but not used. Something like this: https://developers.redhat.com/blog/2021/01/25/introduction-to-containerjfr-jdk-flight-recorder-for-containers

Reply:
Actually, looks like it would be a better idea to add author.job_title under name instead. Not all authors set bio :D